JapaneseFilter/sections/css_extended.txt

```diff
@@ -101,6 +101,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 !
 !
+jp.pinkoi.com#?#.m-card-product:has(.ad)
```

Comment:
If I understand correctly this is out-of-scope as per past cases.

Reply:
I see.